### ISSUE TITLE
Add TypeScript @types support

### ISSUE THREAD

**Issue by jolonf** [thlorenz/sql-escape-string#3]
Add a @types package for TypeScript support or index.d.ts file

**Comment by maneetgoyal** [comment]
Yeah it's such a useful tiny library 👍 , TS support will help a lot

**Comment by jackall3n** [comment]
Given the age of this issue and the fact that there's been no action, I've forked and recreated this package using typescript:

https://github.com/jackall3n/escape-sql-string
https://www.npmjs.com/package/escape-sql-string
